include/arch/imxrt1170.h

```diff
@@ -308,7 +308,7 @@ enum { cti0_err_irq = 17 + 16, cti1_err_irq, core_irq, lpuart1_irq, lpuart2_irq,
 typedef struct {
 	enum { pctl_set = 0, pctl_get } action;
 	enum { pctl_devclock = 0, pctl_iogpr, pctl_iolpsrgpr, pctl_iomux, pctl_iopad, pctl_ioisel, pctl_reboot, pctl_devcache,
-		pctl_lpcg } type;
+		pctl_lpcg, pctl_cleanInvalDCache } type;
```

Comment:
Maybe rename to `pctl_clean_inval_dcache`.

Reply:
It should be camelCase